### DISCUSSION TITLE
Is there an Infura style endpoint for deploying contracts to testnet?

### DISCUSSION THREAD

**Discussion by BarneyChambers** [kcc-community/any-advice-issue#19] [General]
Hi All, 

Very excited to be here to start hacking things out on the testnet. Is there an Infura-style RPC endpoint I can use to deploy contracts onto the KCS testnet? I am looking to add some standard contract for all of us to make use off (Multicall, WETH, etc) 

I have tried with `https://rpc-testnet.kucoin.one` but receiving an error when deploying contracts through this RPC endpoint
`-- only replay-protected (EIP-155) transactions allowed over RPC.`

Thanks for the suggestions and happy hacking!

**Comment by viaweb3** [comment]
hi, maybe you miss the chainId param.  https://ethereum.stackexchange.com/questions/94412/valueerror-code-32000-message-only-replay-protected-eip-155-transac

**Reply by KoffeeBarista** [reply]
I was able to use remix without any issue. If you need an instance of multicall you can use the one I deployed: 0xa12636eAe495c2a5b3D3f566Ce6a19Ea63Cf4e1A

**Reply by BarneyChambers** [reply]
Thanks all, I did have something similar in my truffle config that I have working on Ropsten, Matic, Eth mainnet etc 

    kucointest: {
      provider: () => new HDWalletProvider(mnemonic, `https://rpc-testnet.kucoin.one`),
      network_id: 322,
      confirmations: 2,
      timeoutBlocks: 200,
      skipDryRun: true
    },
    
    I'll keep digging and see what I can find :)
